packages/@okta/vuepress-site/docs/guides/refresh-tokens/main/index.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Refresh access tokens
+excerpt: How to refresh access tokens with Okta
+layout: Guides
+---
+
+This guide teaches you how to refresh access tokens with Okta.
+
+---
+
+**Learning outcomes**
+
+* Understand and set up Refresh token rotation
+* Refresh access tokens
+
+**What you need**
```

Comment:
Remove What you need and Sample code sections

Reply:
@susanharper-okta good call. Sections removed!